manifests/01-operator-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+  annotations:
```

Comment:
For those who missed it, this annotation means "create only, if it does not exist, but leave it alone if it gets updated".